src/engine/attributes/bool.js

```diff
@@ -1,6 +1,11 @@
 
 
-export default function boolApply(val, userVal , op ){
+export default function boolApply(val, userVal, op){
+
+	if(userVal === undefined || userVal === null || userVal.constructor.name != "Boolean") {
```

Comment:
userVal == null is sufficient for catching both null and undefined because _null == undefined_

Reply:
thank you, I will update